lib/index.ts

```diff
@@ -272,6 +288,7 @@ export function main() {
   commander
   .description("Create runtime validator module from TypeScript interfaces")
   .usage("[options] <typescript-file...>")
+  .option("--format <format>", `Format to use for output; options are 'ts', 'js:esm', 'js:cjs'`)
```

Comment:
How about `options are 'ts' (default),` etc?

Reply:
Resolved in 4f68458